documents/validator.go

```diff
@@ -236,6 +289,15 @@ func anchoredValidator(repo anchors.AnchorRepository) Validator {
 			return errors.New("mismatched document roots")
 		}
 
+		tm, err := model.Timestamp()
```

Comment:
`Timestamp` IMO doesn't imply anything on a model. Lets change this to `CreatedAt` ?

Reply:
makes sense

Reply:
unfortunately I would have to defer this as right now coredoc protobuf also has the same name.